src/data/builder-tools.js

```diff
@@ -339,9 +330,19 @@ export const Showcases = [
     getstarted: "/docs/get-started/blockfrost",
     tags: ["favorite", "http", "json", "hosted"],
   },
+  {
+    title: "NMKR Studio",
+    description:
+      "NMKR Studio removes the complexity of building Blockchain products. Making it easy to launch NFT collections",
+    preview: require("./builder-tools/nmkr.png"),
+    website: "https://www.nmkr.io/studio",
+    getstarted: "https://studio.nmkr.io/register/",
```

Comment:
```suggestion
    getstarted: "https://docs.nmkr.io/NMKR-Studio/introduction",
```
`getstarted:` is intended to be a starting point for _learning_ about a tool, not for "getting started" with a _subscription_ to it 😜 This change seemed to be the best for new users in my opinion 🤔

Reply:
Thanks for the quick response. I did the changes as suggested. I also noticed that NFT-MAKER under showcases is still under the old name, so I updated the naming and link. (former https://www.nft-maker.io/ now redirects to https://nmkr.io anyway, but to make it consistent and reflect the naming convention)

Reply:
Sorry it seems as git showing it as different, that is however only because it is the original formatting (git compares it to the previous commit)